docs/source/scripts/environment_besca.yml

```diff
@@ -1,188 +1,157 @@
-- xorg-xextproto=7.3.0=h14c3975_1002
-- xorg-xproto=7.0.31=h14c3975_1007
-- xz=5.2.4=h14c3975_1001
-- zlib=1.2.11=h14c3975_1004
-- pip:
-  - alabaster==0.7.12
-  - alembic==1.0.8
-  - anndata==0.6.22
-  - annoy==1.15.1
-  - atomicwrites==1.3.0
-  - attrs==19.1.0
-  - Babel==2.6.0
-  - backcall==0.1.0
-  - bbknn==1.3.1
-  - biothings-client==0.2.0
-  - bleach==3.1.0
-  - bottle==0.12.13
-  - certifi==2018.11.29
-  - cffi==1.12.2
-  - chardet==3.0.4
-  - Click==7.0
-  - cloudpickle==0.8.0
-  - cycler==0.10.0
-  - Cython==0.29.6
-  - dask==1.1.3
-  - decorator==4.3.2
-  - defusedxml==0.5.0
-  - distributed==1.18.0
-  - docopt==0.6.2
-  - docutils==0.14
-  - entrypoints==0.3
-  - h5py==2.9.0
-  - HeapDict==1.0.0
-  - idna==2.8
-  - imagesize==1.1.0
-  - ipykernel==5.1.0
-  - ipython==6.2.1
-  - ipython-genutils==0.2.0
-  - ipywidgets==7.4.2
-  - jedi==0.13.3
-  - Jinja2==2.10
-  - joblib==0.13.2
-  - jsonschema==3.0.1
-  - jupyter==1.0.0
-  - jupyter-client==5.2.4
-  - jupyter-console==5.2.0
-  - jupyter-core==4.4.0
-  - kiwisolver==1.0.1
-  - llvmlite==0.27.1
-  - louvain==0.6.1
-  - Mako==1.0.7
-  - MarkupSafe==1.1.1
-  - matplotlib==3.0.3
-  - mistune==0.8.4
-  - mnnpy==0.1.9.5
-  - more-itertools==6.0.0
-  - msgpack-python==0.4.8
-  - MulticoreTSNE==0.1
-  - mygene==3.1.0
-  - natsort==6.0.0
-  - nbclean==0.3.2
-  - nbconvert==5.4.1
-  - nbformat==4.4.0
-  - nbgrader==0.5.5
-  - nbsphinx==0.4.2
-  - networkx==2.2
-  - nose==1.3.7
-  - notebook==5.7.5
-  - numba==0.42.1
-  - numexpr==2.6.9
-  - numpy==1.16.2
-  - packaging==19.0
-  - pandas==0.24.1
-  - pandocfilters==1.4.2
-  - parso==0.3.4
-  - patsy==0.5.1
-  - pexpect==4.6.0
-  - pickleshare==0.7.5
-  - Pillow==5.3.0
-  - plotly==3.7.1
-  - pluggy==0.9.0
-  - prometheus-client==0.6.0
-  - prompt-toolkit==1.0.15
-  - psutil==5.5.1
-  - ptyprocess==0.6.0
-  - py==1.8.0
-  - pycairo==1.18.0
-  - pycparser==2.19
-  - Pygments==2.3.1
-  - pymongo==3.6.1
-  - pyparsing==2.3.1
-  - pyrsistent==0.14.11
-  - pytest==4.3.0
-  - python-dateutil==2.8.0
-  - python-editor==1.0.4
-  - python-igraph==0.7.1.post6
-  - pytz==2018.9
-  - PyYAML==3.13
-  - pyzmq==18.0.0
-  - qtconsole==4.4.3
-  - requests==2.21.0
-  - retrying==1.3.3
-  - rpy2==3.0.1
-  - runipy==0.1.5
-  - scanpy==1.4.1
-  - scikit-learn==0.20.3
-  - scipy==1.1.0
-  - seaborn==0.9.0
-  - Send2Trash==1.5.0
-  - simplegeneric==0.8.1
-  - six==1.12.0
-  - snowballstemmer==1.2.1
-  - sortedcontainers==2.1.0
-  - Sphinx==1.8.4
-  - sphinx-autodoc-annotation==1.0.post1
-  - sphinx-automodapi==0.10
-  - sphinx-gallery==0.2.0
-  - sphinx-rtd-theme==0.4.3
-  - sphinxcontrib-websupport==1.1.0
-  - SQLAlchemy==1.3.0
-  - statsmodels==0.9.0
-  - tables==3.4.4
-  - tblib==1.3.2
-  - terminado==0.8.1
-  - testpath==0.4.2
-  - toolz==0.9.0
-  - tornado==6.0.1
-  - tqdm==4.31.1
-  - traitlets==4.3.2
-  - urllib3==1.24.1
-  - wcwidth==0.1.7
-  - webencodings==0.5.1
-  - widgetsnbextension==3.4.2
-  - zict==0.1.2
+  - _libgcc_mutex=0.1=conda_forge
```

Comment:
I think these are still version numbers specific to linux. See issue #65. Potentially regenerate with --no-builds to solve.